<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 16pt &quot;JetBrains Mono&quot;;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="mainGroup">
     <property name="styleSheet">
      <string notr="true">QGroupBox#mainGroup {
	font:600 italic 25pt &quot;JetBrains Mono&quot;;
	color:rgb(0, 0, 127);
	border-color:rgb(0, 0, 127);
}

QGroupBox#mainGroup::title {
	margin-left:-1ex;
}

QGroupBox {
	border-width:2px;
	border-style:solid;
	border-radius:10px;
	border-color:gray;
	margin-top:1.6ex;
}

QGroupBox::title{
	subcontrol-origin:margin;
	subcontrol-position:top centorl;
	padding:0 -4px;
	margin-left:0ex;
}
</string>
     </property>
     <property name="title">
      <string>Game Record Setting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="recordLayout">
        <item>
         <widget class="QGroupBox" name="recordGroup_1">
          <property name="title">
           <string>Game Record</string>
          </property>
          <layout class="QHBoxLayout" name="recordLayout_1">
           <item>
            <widget class="QRadioButton" name="gameOn">
             <property name="text">
              <string>On</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="gameOff">
             <property name="text">
              <string>Off</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="recordGroup_2">
          <property name="title">
           <string>Event Record</string>
          </property>
          <layout class="QHBoxLayout" name="recordLayout_2">
           <item>
            <widget class="QRadioButton" name="eventOn">
             <property name="text">
              <string>On</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="eventOff">
             <property name="text">
              <string>Off</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gameTypeConbo">
             <property name="insertPolicy">
              <enum>QComboBox::InsertAtBottom</enum>
             </property>
             <item>
              <property name="text">
               <string>HEX</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="OptionGroup">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Compertition Option</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="OptionLayout" stretch="7,6,6,6">
         <item>
          <layout class="QHBoxLayout" name="optionLayout_1">
           <item>
            <widget class="QLabel" name="optionLabel_1">
             <property name="text">
              <string>Competition:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="optionEdit_1"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="optionLayout_2">
           <item>
            <widget class="QLabel" name="optionLabel_2">
             <property name="text">
              <string>   Location:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="optionEdit_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="optionLayout_3">
           <item>
            <widget class="QLabel" name="optionLabel_3">
             <property name="text">
              <string> First Team:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="optionEdit_3"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="optionLayout_4">
           <item>
            <widget class="QLabel" name="optionLabel_4">
             <property name="text">
              <string>Second Team:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="optionEdit_4"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
